environment.yml

```diff
@@ -36,6 +36,7 @@ dependencies:
 - pre-commit>=1.14.4
 - pyyaml>=5.1.2
 - requests>=2.22.0
+- cytoolz
```

Comment:
@jiata: I added the 04 notebook to get some early feedback. It's unfinished, but I am planning to clean it up a little and correct typos and would then also submit to staging (even without the speed/accuracy tests done, but showing that we can indeed reproduce the mAP number from torchvision)

Reply:
ps. leaving a comment here since I cannot leave a comment in the 04 notebook itself (too big for github)